lms/djangoapps/teams/models.py

```diff
@@ -75,10 +75,16 @@ def add_user(self, user):
             raise NotEnrolledInCourseForTeam
         if CourseTeamMembership.user_in_team_for_course(user, self.course_id):
             raise AlreadyOnTeamInCourse
-        return CourseTeamMembership.objects.create(
+        membership = CourseTeamMembership.objects.create(
             user=user,
             team=self
         )
+        return membership
```

Comment:
Nit: This is an odd change, just because it doesn't really do anything differently?